src/main/java/seedu/storemando/model/item/QuantityComparator.java

```diff
@@ -0,0 +1,17 @@
+package seedu.storemando.model.item;
+
+import java.util.Comparator;
+
+public class QuantityComparator implements Comparator<Item> {
+
+    @Override
+    public int compare(Item firstItem, Item secondItem) {
```

Comment:
I get what you are trying to do here. But your code here seems like you are comparing two items instead of comparing the item's quantity. 

For me personally, I feel like this entire code should be abstracted to the item class and name it as quantityPriorityComparison.

What do others think?

Reply:
Agree with this, but if its functional we can leave this to v1.3 as well. :)